stdlib/public/SDK/Foundation/IndexSet.swift

```diff
@@ -354,6 +354,11 @@ public struct IndexSet : ReferenceConvertible, Equatable, BidirectionalCollectio
             return i..<i
         }
         
+        guard let first = first, let last = last else {
+            let i = _index(ofInteger: 0)
+            return i..<i
+        }
+
```

Comment:
let's just put this guard statement up there with the range.isEmpty check to avoid copy/pasting the code.